packages/react-i18n/src/i18n-locale.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable */
```

Comment:
This is duplicated from the upstream PR. It would come from the package and not exist here.